docs/topics/api/reviewers.rst

```diff
@@ -335,14 +335,14 @@ These endpoints allow you to draft comments that can be submitted through the re
 
     :>json int id: The id of the draft comment object.
     :>json string comment: The comment that is being drafted as part of a review. Specific to a line in a file.
-    :>json string filename: The filename a specific comment is related to.
-    :>json int lineno: The line number a specific comment is related to. Please make sure that in case of comments for git diffs, that the `lineno` used here belongs to the file in the version that belongs to `version_id` and not it's parent.
+    :>json string|null filename: The filename a specific comment is related to.
+    :>json int|null lineno: The line number a specific comment is related to. Please make sure that in case of comments for git diffs, that the `lineno` used here belongs to the file in the version that belongs to `version_id` and not it's parent.
```

Comment:
similarly here.  Is it not being related to a specific file the only reason, or will it be null if it relates to a line that doesn't exist in the parent file from a diff too?

Reply:
Fixed in 540e0af